iceoryx_posh/include/iceoryx_posh/gateway/toml_gateway_config_parser.hpp

```diff
@@ -51,17 +51,17 @@ static constexpr const char GATEWAY_CONFIG_SERVICE_EVENT_NAME[] = "event";
 class TomlGatewayConfigParser
 {
   public:
-    static iox::cxx::expected<GatewayConfig, TomlGatewayConfigParseError> parse();
-    static iox::cxx::expected<GatewayConfig, TomlGatewayConfigParseError> parse(ConfigFilePathString_t path);
+    static cxx::expected<GatewayConfig, TomlGatewayConfigParseError> parse();
+    static cxx::expected<GatewayConfig, TomlGatewayConfigParseError> parse(ConfigFilePathString_t path);
 
   protected:
-    static iox::cxx::expected<TomlGatewayConfigParseError> validate(const cpptoml::table& parsedToml) noexcept;
+    static cxx::expected<TomlGatewayConfigParseError> validate(const cpptoml::table& parsedToml) noexcept;
 
   private:
     static bool hasInvalidCharacter(std::string s) noexcept;
```

Comment:
I know you did not touch this method but could you please make the argument a const reference and if possible use `cxx::string`.
```
static bool hasInvalidCharacter(const cxx::string& s) noexcept;
```